src/main/java/it/gov/pagopa/afm/calculator/Application.java

```diff
@@ -5,7 +5,7 @@
 import org.springframework.cache.annotation.EnableCaching;
 
 @SpringBootApplication
-@EnableCaching
+//@EnableCaching
```

Comment:
Maybe we can restore caching before merging into main?

Reply:
cache enabled! 👍